### ISSUE TITLE
Change wording of Security Setting

### ISSUE THREAD

**Issue by gillerby** [tutao/tutanota#1679]
Currently, in Global Settings, the Security one reads: "Save encrypted IP address in sessions and audit log"

This is ambiguous, as at first I thought it meant whether I wanted to save IPs in an encrypted manner or without encryption, not if I wanted to save Ips (encrypted) or not at all.

Please reword it to something like: "Save IPs in audit log and session? (They will be fully encrypted)"

**Comment by gillerby** [comment]
Bump? This should be easy to implement.

**Comment by gillerby** [comment]
?

**Comment by jowlo** [comment]
Hi, i changed the label to
> Enable saving of IP addresses in sessions and audit log. IP addresses are stored encrypted.

I hope it's harder to misunderstand. This will show up in the app on the next language update, should not take too long.

**Comment by vaf-hub** [comment]
Finding during testin:

The translation key has not been updated on master. Did you run the update-translations script @charlag ?
https://github.com/tutao/tutanota/blob/ca5c57d6d84ba1ee2f2b748cb4ca5c1ed3b38dde/src/translations/en.js#L1034

Also, I checked phrase and noticed that the German translation keys have not been changed at all. I think we should always do that. (Just added the German translations in phrase.) Sometimes it is best to delete a key and recreate it to force an update of the translation. Is there a way to unverify all translations of a specific key in phrase?

**Comment by jowlo** [comment]
A language update would be nice for #2861 too as there are only `de` `de_sie` and `en` on `master` but quite a few translations already in phrase.

**Comment by charlag** [comment]
I didn't run an update, I can put it into another release if we don't have any other reasons to rebuild.
I think when you change English it unverifies the rest?

**Comment by vaf-hub** [comment]
Oh that's true. I thought it did not work, when I tried it, but apparently it does unverify all other translations. Don't know what went wrong in my first attempt.
